samples/net/sockets/echo_async/src/socket_echo.c

```diff
@@ -102,19 +102,23 @@ int main(void)
 {
```

Comment:
You need to split the "samples: sockets: echo_async: Add support for cc3220sf_launchxl " patch. The board specific changes should go to a patch of its own.

Reply:
Thanks, I made the split.